NetworkConfigurations/NetworkAddressingAndFilteringSubCategories/Stateful-NAT.md

```diff
@@ -0,0 +1,139 @@
+---
+title: Stateful Network Address Translation
+grand_parent: Network Configuration
+parent: Network Addressing and Filtering
+nav_order: 5
+layout: default
+---
+
+# Stateful Network Address Translation (NAT)
+
+Stateful Network Address Translation (NAT), also known as dynamic NAT, facilitates the mapping of multiple private IP addresses to public addresses.
+
+- It was introduced in the [Beeblebrox Release (v2)](https://github.com/dentproject/dentOS/releases/tag/v2.0) of DENT.
+
+Configuration of stateful NAT involves employing the `tc iproute2` tool alongside the `tc-ct` action. For dynamic NAT entry offloading, integrate the `tc-ct` action with the flower filter rule and ingress qdisc.
+
+### Example Configuration:
```

Comment:
I see this example configuration, but I do not know if you've tested it. If you did, please add a section on how you verified this config.